### ISSUE TITLE
Remove changelog from readme.txt

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amp-wp#1871]
Background: I want to make it easier to translate this plugin so more people from around the world can learn about it.

Plugins hosted on WordPress.org can be fully translated by polyglots. This applies to the code _and_ the `readme.txt`.

Translated readmes are available on Rosetta pages like https://es.wordpress.org/plugins/amp/ and https://tw.wordpress.org/plugins/amp/.

The problem right now is that people have to translate the complete changelog of the plugin in order to reach 100% for the translation. Some people actually try to translate changelog entries, others simply copy and paste the English text.

In order to remove this burden from polyglots, the changelog section should just contain a link to the living changelog on GitHub, either https://github.com/ampproject/amp-wp/releases or preferably an actual `CHANGELOG.md` file.

Examples of plugins already doing this:

https://wordpress.org/plugins/preferred-languages/#developers
https://wordpress.org/plugins/query-monitor/#developers

While we're at it, I would also update `README.md` to point to `CHANGELOG.md`. Right now, it's basically a Markdown copy of `readme.txt`, which I think doesn't make much sense. I'd love to see a `README.md` file focused at developers contributing to the project on GitHub.

So, in summary:

* Remove changelog from `readme.txt` and point to GitHub instead
* Create separate changelog file on GitHub for easier referencing
* Update `readme.md` to make it more accessible to GitHub audience

Hope this makes sense. Happy to create a PR to kick this off.

**Comment by westonruter** [comment]
> In order to remove this burden from polyglots, the changelog section should just contain a link to the living changelog on GitHub, either https://github.com/ampproject/amp-wp/releases or preferably an actual `CHANGELOG.md` file.

Sounds good to me. I like the idea of linking to the releases page on GitHub as that will eliminate maintenance of having to update `CHANGELOG.md` as well. We've been essentially putting the changelog in the release notes anyway.

> Right now, it's basically a Markdown copy of `readme.txt`, which I think doesn't make much sense.

That's right. The `readme.md` is automatically generated from `readme.txt` via `grunt readme`. The intention was to eliminate having to maintain two separate readmes.

> I'd love to see a `README.md` file focused at developers contributing to the project on GitHub.

Isn't this what the `contributing.md` is for?

**Comment by swissspidy** [comment]
> That's right. The `readme.md` is automatically generated from `readme.txt` via `grunt readme`. The intention was to eliminate having to maintain two separate readmes.
> Isn't this what the `contributing.md` is for?

IMO these they target two different audiences and therefore should have different content.

For `readme.md` I would probably just leave the badges, a short intro and a link to `contributing.md`. Everything else is explained on amp-wp.org or WordPress.org, so we can link to these as well.

**Comment by westonruter** [comment]
Let's start with just the changelog and then address the readme later.

**Comment by westonruter** [comment]
Changelog was removed.
